### PR TITLE
Small cleanup in root package.json

### PR DESCRIPTION
**Pull request by Andarist**
babel@6 is still used:
- here https://github.com/emotion-js/emotion/blob/b81808d127853aed6aeb6620949805afc03b58e6/packages/primitives/test/no-babel/.babelrc
- and here https://github.com/emotion-js/emotion/blob/b81808d127853aed6aeb6620949805afc03b58e6/packages/create-emotion/test/.babelrc

Should I just update both places to use babel@7? Are those babelrc files even used now when we have babel.config.js in the root?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "private": true,
   "name": "emotion-monorepo",
-  "version": "7.0.13",
```
[the remark this answers is not in the document]
Comment (Andarist) [reply package.json]: hm, maybe just let's see if any tool will complain about it missing?

File: package.json [hunk 2]
```diff
@@ -207,19 +202,19 @@
...
       "threshold": "7 Kb"
     },
     {
-      "path": "./packages/react-emotion/dist/emotion.umd.min.js",
+      "path": "./packages/styled-base/dist/styled-base.umd.min.js",
       "threshold": "9.7 Kb"
     },
     {
-      "path": "./packages/core/dist/index.min.js",
+      "path": "./packages/core/dist/core.min.js",
```
[the remark this answers is not in the document]
Comment (Andarist) [reply package.json]: good catch

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mitchellhamilton) [reply package.json]: iirc vs code complains about it and i think bolt might as well. It doesn't really harm anything so why not leave it?

[the remark this answers is not in the document]
Comment (Andarist) [reply package.json]: ok, i have brought it back

**Comment by Andarist** [comment]
Should be ready to merge now - please do the review again :)

**Comment by Andarist** [comment]
friendly 🏓 @mitchellhamilton

**Pull request merged**